ophyd/ophydobj.py

```diff
@@ -75,6 +99,40 @@ def finished_cb(self, cb):
         else:
             self._cb = cb
 
+    def wait(self, timeout=30.0, *, poll_rate=0.05):
```

Comment:
Maybe the default should be `None` here also, lacking a universally appropriate default. If this is mainly for interactive use, the user still has ^C.